buttercup.el

```diff
@@ -675,6 +675,7 @@ See also `buttercup-define-matcher'."
 (cl-defstruct (buttercup-suite (:include buttercup-suite-or-spec))
   ;; Any children of this suite, both suites and specs
   children
+  environment
```

Comment:
This is probably unnecessary, we can track the environment in the `buttercup--current-env` variable (seems like a duplicate)